its/ruling/src/test/resources/expected_extended/nltk/python-S930.json

```diff
@@ -1,7 +1,4 @@
 {
-'nltk:nltk/ccg/lexicon.py':[
-77,
```

Comment:
This symbol was only defined in our Python 2 stubs. It's fair that we lose the issue in this context.
Not sure whether it was TP/FP though.